input/fsh/SD_TumorSize.fsh

```diff
@@ -76,6 +84,17 @@ Description:  "Identifies a tumor. Whenever possible, a single resource conformi
 * location 1..1 MS // Tumor is meaningless without a location; parent profile is 0..1
 * patient only Reference(CancerPatient)
 
+// Use locationQualifier to identify a multifocal tumor, in addition to modifying body site
+* locationQualifier from http://hl7.org/fhir/ValueSet/bodystructure-relative-location (example)
```

Comment:
Why not `LocationQualifierVS (required)`? We do that for the location of the PrimaryCancerCondition. Why wouldn't we do that here?